### PR TITLE
normalizing saturations

### PR DESCRIPTION
**Pull request by hnil**
In progress. Aim at normalizing the saturations to avoid the danger of haveing bad results. Initial test give worse performance on Norne, also norne output still have negative oil saturation in water cells.

### CODE REVIEW

**Comment by hnil** [comment]
This is work in progress.

**Comment by hnil** [comment]
With the option it the only real change is that it do not project the solvent saturation in the default setting. My opinion is that all saturation should be treated consistently if we do not have a very specific reason

**Comment by bska** [comment]
jenkins build this please

**Comment by bska** [comment]
jenkins build this please

File: opm/models/blackoil/blackoilprimaryvariables.hh [hunk 1]
```diff
@@ -363,8 +363,8 @@ public:
         if (primaryVarsMeaning() == OnePhase_p){
             return false;
         }
-        Scalar eps_sat=eps;// eps to represent near the phase disapear 
-        Scalar eps_dis=eps*100;//eps to represent near saturated
+        Scalar eps_sat=0.0//eps;// eps to represent near the phase disapear 
+        Scalar eps_dis=0.0//eps*100;//eps to represent near saturated
```
Comment (bska) [inline opm/models/blackoil/blackoilprimaryvariables.hh:367]: You need to reinsert the semicolons here.

**Comment by hnil** [comment]
I will close this pullrequest to get one more inline with OPM policy: the new will be https://github.com/OPM/opm-models/pull/610

**Pull request closed without merging**